app/assets/stylesheets/form.scss

```diff
@@ -0,0 +1,44 @@
+@mixin lato-book { font-family: 'Lato', sans-serif; font-weight: 300; }
+$white: #ffffff;
+
+.form {
+  margin: auto;
+  width: 80%;
+  max-width: 750px;
+  background: $white;
+
+  .button {
+    margin-top: 10px;
+    width: 100%;
+    display: inline;
+  };
+}
+
+.form .row {
+  padding: 0 2.5em;
+  max-width: none;
+  &.header {
+    padding: 1.5em 2em;
```

Comment:
Properties should be ordered background, -moz-background-size, -o-background-size, -webkit-background-size, background-size, border-bottom, color, padding